themes/default/content/registry/packages/azure/installation-configuration.md

```diff
@@ -19,27 +19,31 @@ The Azure Classic provider is available as a package in all Pulumi languages:
 * Go: [`github.com/pulumi/pulumi-azure/sdk/v4/go/azure`](https://github.com/pulumi/pulumi-azure)
 * .NET: [`Pulumi.Azure`](https://www.nuget.org/packages/Pulumi.Azure)
 
-## Setup
+## Credentials
 
-Pulumi can authenticate to Azure using a Service Principal or the Azure CLI.
+Pulumi can authenticate to Azure via the Azure CLI or using a Service Principal.
 
 If you're running the Pulumi CLI locally, in a developer scenario, we recommend using the Azure CLI. For team environments, particularly in CI, we recommend using a Service Principal.
 
-> **Note:** Authenticating using the CLI will not work for Service Principal logins (e.g.,
-> `az login --service-principal`).  For such cases, authenticate using the Service Principal method instead.
+{{% notes type="info" %}}
+Authenticating using the CLI will not work for Service Principal logins (e.g.,
+`az login --service-principal`).  For such cases, authenticate using the Service Principal method instead.
+{{% /notes %}}
 
-### Option 1: Use the CLI
+### Authenticate using the CLI
 
-Simply login to the Azure CLI and Pulumi will automatically use your credentials:
+Log in to the Azure CLI and Pulumi will automatically use your credentials:
 
 ```bash
 $ az login
-The default web browser has been opened at https://login.microsoftonline.com/common/oauth2/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
+A web browser has been opened at https://login.microsoftonline.com/organizations/oauth2/v2.0/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
 ```
 
-Do as instructed to login.  After completed, `az login` will return and you are ready to go.
+Do as instructed to log in.  After completed, `az login` will return and you are ready to go.
 
-> **Note:** If you're using Government, China, or German Clouds, you'll need to configure the Azure CLI to work with that cloud.  Do so by running `az cloud set --name <Cloud>`, where `<Cloud>` is one of `AzureUSGovernment`, `AzureChinaCloud`, or `AzureGermanCloud`.
+{{% notes type="info" %}}
+If you're using Government, China, or German Clouds, you'll need to configure the Azure CLI to work with that cloud.  Do so by running `az cloud set --name <Cloud>`, where `<Cloud>` is one of `AzureUSGovernment`, `AzureChinaCloud`, or `AzureGermanCloud`.
```

Comment:
you could remove the German cloud while you're touching this, it's gone: https://docs.microsoft.com/en-us/previous-versions/azure/germany/germany-welcome

Reply:
done! thank you for catching this!